maintainer_guide.md

```diff
@@ -0,0 +1,75 @@
+## Torchvision maintainers guide
+
+This document aims at documenting user-facing policies / principles used when
+developing and maintaining torchvision. Other maintainer info (e.g. release
+process) can be found in the internal wiki.
+
+### What is public and what is private?
+
+For the Python API, torchvision largely follows the [torch core
```

Comment:
I'm never sure how we should call core: torch or PyTorch?

Reply:
over here it's usually "torch core" or "core", but "PyTorch" isn't ambiguous either. Happy with whatever you prefer

Reply:
In that case, I'm in favor of "torch" since with that we don't have a disconnect in the naming scheme when talking about "torchvision". (I really hate that in `conda install pytorch torchvision -c pytorch`)

Reply:
Sorry to be pedantic but  "torch" may refer to the original Lua torch ? https://en.wikipedia.org/wiki/Torch_(machine_learning)